pl_examples/domain_templates/semantic_segmentation.py

```diff
@@ -17,6 +17,29 @@
 DEFAULT_VALID_LABELS = (7, 8, 11, 12, 13, 17, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 31, 32, 33)
 
 
+def _create_synth_Cityscapes_dataset(path_dir):
+    """Create synthetic dataset with random images, just to simulate that the dataset have been already downloaded."""
+    non_existing_citites = ['dummy_kitti_1', 'dummy_kitti_2']
+    fine_labels_dir = Path(path_dir) / 'gtFine'
+    images_dir = Path(path_dir) / 'leftImg8bit'
+    dataset_splits = ['train', 'val', 'test']
+
+    for split in dataset_splits:
+        for city in non_existing_citites:
+            (images_dir / split / city).mkdir(parents=True)
+            (fine_labels_dir / split / city).mkdir(parents=True)
+            base_name = str(uuid.uuid4())
+            image_name = f'{base_name}_leftImg8bit.png'
+            instance_target_name = f'{base_name}_gtFine_instanceIds.png'
+            semantic_target_name = f'{base_name}_gtFine_labelIds.png'
+            Image.new('RGB', (2048, 1024)).save(
```

Comment:
Do you need the image to be 2048 * 1024 ?

Reply:
pls ignore this func, it is till very WIP :D